src/components/Buttons/__tests__/RadioGroup.spec.js

```diff
@@ -14,26 +14,26 @@ describe('RadioGroup', () => {
       </RadioGroup>
     )
 
-    wrapper.children().forEach(child => {
+    wrapper.find('Radio').forEach(child => {
       expect(child.prop('isSelected')).toBe(false)
     })
   })
 
   it('only one radio button can be selected at a time', () => {
     const wrapper = mount(
       <RadioGroup name="radioTest">
-        <Radio id="1" value="val1">Value 1</Radio>
-        <Radio id="2" value="val2">Value 2</Radio>
-        <Radio id="3" value="val3">Value 3</Radio>
+        <Radio id="A" className="radio" value="val1">Value 1</Radio>
```

Comment:
using `className` because Enzyme now wraps everything (even after `.find()`) with a wrapper node, making `.children()` return the wrapper instead of expected nodes.

Reply:
see [the enzyme migration docs](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#children-now-has-slightly-different-meaning) regarding the change in `.children()` results